### PR TITLE
fix(docs): add undocumented groups field to deployment object

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

**Review by alfrunes — APPROVED** [review]

File: docs/management_api.yml [hunk 1]
```diff
@@ -1597,6 +1597,13 @@ definitions:
         description: An array of artifact's identifiers.
         items:
           type: string
+      groups:
+        type: array
+        description: |
+            An array of groups the devices targeted by the deployment belongs to.
```
Comment (tranchitella) [inline docs/management_api.yml:1603]:
```suggestion
            An array of groups the devices targeted by the deployment belong to.
```

File: docs/management_api.yml [hunk 2]
```diff
@@ -1597,6 +1597,13 @@ definitions:
...
+            Available only if user created deployment for a group or for a single device (if the device was in a static group).
```
Comment (tranchitella) [inline docs/management_api.yml:1604]:
```suggestion
            Available only if the user created the deployment for a group or a single device (if the device was in a static group).
```

**Review by tranchitella — APPROVED** [review]
Pre-approving, the change is trivial.

File: docs/management_api.yml (hunk 1 above)
Comment (kjaskiewiczz) [reply docs/management_api.yml]: fixed

File: docs/management_api.yml (hunk 2 above)
Comment (kjaskiewiczz) [reply docs/management_api.yml]: fixed

**Pull request merged**